app/assets/javascripts/main.js

```diff
@@ -79,3 +79,4 @@ var type = ""
     }
 
   mapLeaflet.scrollWheelZoom.disable();
+// ugly JS FTW
```

Comment:
@neight-allen any thoughts on how to improve the promises in this file to speed up loading?